src/utils/web3.utils.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * Should be called to get hex representation (prefixed by 0x) of ascii string
+ * ref: https://github.com/ChainSafe/web3.js/blob/63e73bb3f8d4720edcc31441c3b613758ad96cec/packages/web3-utils/src/index.js
+ * @method asciiToHex
+ * @param {String} str
+ * @returns {String} hex representation of input string
+ */
+ export const asciiToHex = (str: string): string => {
+  if (!str) return '0x00';
+  let hex = '';
+  for (let i = 0; i < str.length; i++) {
+    const code = str.charCodeAt(i);
+    const n = code.toString(16);
+    hex += n.length < 2 ? '0' + n : n;
+  }
+
+  return '0x' + hex;
+};
```

Comment:
`formatBytes32String` from Ethers does the same thing: https://docs.ethers.io/v4/api-utils.html#bytes32-strings
It pads zeroes, but it should still work in the callsites.